sql/2021/ecommerce/ecomm_vendors_covid_growth.sql

```diff
@@ -0,0 +1,38 @@
+#standardSQL
+# 13_04: Timeseries to show eCommerce vendors growth acceleration due to Covid-19
+# Excluding apps which are not eCommerce platforms/vendors themselves but are used to identify eCommerce sites. These are signals added in Wappalyzer in 2020 to get better idea on % of eCommerce sites but these are not relevant for vendor % market share analysis
+# Limiting to top 5000 records to continue further analysis in Google Sheets. Using HAVING clauses based on 'pct' results in missing data for certain months
+SELECT
+  IF(ENDS_WITH(_TABLE_SUFFIX, '_desktop'), 'desktop', 'mobile') AS client,
+  app,
+  COUNT(DISTINCT url) AS freq,
+  total,
+  COUNT(DISTINCT url) / total AS pct,
+  LEFT(_TABLE_SUFFIX, 4) AS year,
+  SUBSTR(_TABLE_SUFFIX, 6, 2) AS month
+FROM
+  `httparchive.technologies.*`
```

Comment:
Editorially, I think we should focus on 2021 data. The previous query compares 2021 and 2020 year-over-year data, which is totally fine, but here it seems excessive to analyze the entire dataset.

cc @bobbyshaw